state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -423,14 +438,11 @@ pub enum PalletConfigUpdate<T: Config> {
 	/// Set the broker bond. This is the amount of FLIP that must be bonded to open a private
 	/// broker channel. The funds are getting freed when the channel is closed.
 	SetBrokerBond { bond: T::Amount },
-	/// Set the minimum fee in USDC paid per swap request
-	SetMinimumNetworkFee { min_fee: AssetAmount },
-	/// Set the network fee in USDC that will be used just for internal swaps (credit on-chain
-	/// swaps)
-	SetInternalSwapNetworkFee { rate: Permill },
-	/// Set the minimum network fee in USDC that will be used just for internal swaps (credit
-	/// on-chain swaps)
-	SetInternalSwapMinimumNetworkFee { min_fee: AssetAmount },
+	/// Set the network fee rate and minimum in USDC
+	SetNetworkFee { rate: Permill, minimum: AssetAmount },
```

Comment:
Might be annoying that you have to provide both values when you only want to change one (also comes with a risk of unintentionally changing the other). What if we made them optional and only apply the change for provided fields?